### PR TITLE
migrate code from googleapis/nodejs-cloud-container

### PR DESCRIPTION
**Pull request by grayside**
- new generated code, updating version to 0.8.0
- Better readme and quickstart (#4)
- Upgrade repo-tools and regenerate scaffolding. (#19)
- v0.2.0 (#20)
- chore: lock files maintenance (#24)
- chore: lock files maintenance (#26)
- chore: lock files maintenance (#29)
- chore: update sample lockfiles (#38)
- chore(deps): update dependency @google-cloud/nodejs-repo-tools to v2.3.0 (#41)
- chore(deps): lock file maintenance (#44)
- chore(deps): lock file maintenance (#47)
- fix: drop support for node.js 4.x and 9.x (#46)
- chore(deps): lock file maintenance (#48)
- chore(deps): lock file maintenance (#49)
- chore(deps): lock file maintenance (#51)
- chore(deps): lock file maintenance (#54)
- chore: require node 8 for samples (#56)
- chore(deps): lock file maintenance (#57)
- chore(deps): lock file maintenance (#61)
- chore: ignore package-lock.json (#63)
- Remove unused dependencies (#72)
- chore(deps): update dependency @google-cloud/nodejs-repo-tools to v3 (#113)
- docs(samples): updated samples code to use async await (#122)
- Release v0.3.0 (#141)
- test: add sample tests (#143)
- chore(deps): update dependency mocha to v6
- Release v0.3.1 (#168)
- refactor: use execSync for tests (#176)
- refactor: wrap execSync with encoding: utf-8 (#177)
- chore: release 1.0.0 (#193)
- refactor: use repo-metadata to generate readme (#196)
- chore: release 1.1.0 (#204)
- chore: release 1.1.1 (#206)
- chore: release 1.1.2 (#209)
- chore: release 1.1.3 (#211)
- chore: release 1.1.4 (#222)
- chore: release 1.2.0 (#225)
- chore: release 1.3.1 (#241)
- chore: release 1.4.0 (#248)
- chore: update license headers
- refactor: use explicit mocha imports
- chore(deps): update dependency mocha to v7 (#269)
- chore: release 1.5.0
- chore: release 1.6.0 (#274)
- chore: release 1.6.1 (#278)
- chore: release 1.6.2 (#287)
- chore: release 1.7.0 (#296)
- chore: release 2.0.0 (#319)
- chore(deps): update dependency mocha to v8 (#349)
- chore: release 2.1.0 (#348)
- chore: release 2.1.1 (#359)
- chore: release 2.1.2 (#399)
- chore: release 2.2.0 (#402)
- chore: release 2.2.1 (#432)
- chore: release 2.2.2 (#442)
- chore: release 2.3.0 (#447)
- chore: release 2.3.1 (#456)
- chore: release 2.3.2 (#461)
- chore: release 2.4.0 (#465)
- chore: release 2.4.1 (#467)
- chore: release 2.4.2 (#469)
- chore: release 2.4.3 (#478)
- chore: release 2.4.4 (#479)
- chore: release 2.5.0 (#481)
- docs(samples): add auto-generated Node samples (#488)
- chore: release 2.6.0 (#498)
- docs(samples): add usage samples to show handling of LRO response Operation (#519)
- chore(main): release 3.0.0 (#511)
- chore(main): release 3.0.1 (#529)
- build!: update library to use Node 12 (#539)
- chore(main): release 4.0.0 (#540)
- chore(main): release 4.0.1 (#542)
- chore(main): release 4.1.0 (#550)
- chore(main): release 4.1.1 (#559)
- chore(main): release 4.1.2 (#565)
- fix(deps): update dependency uuid to v9 (#564)
- chore(main): release 4.1.3 (#566)
- chore(main): release 4.2.0 (#569)
- chore(main): release 4.3.0 (#579)
- chore(main): release 4.4.0 (#584)

### CODE REVIEW

File: container/snippets/quickstart.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    const [response] = await client.listClusters(request);
+    console.log('Clusters: ', response);
+  }
+  quickstart();
+  // [END container_quickstart]
+}
+
+main().catch(console.error);
+// [END gke_list_cluster]
```
Comment (ace-n) [inline container/snippets/quickstart.js:41]: This region tag seems unused

**Review by ace-n — APPROVED** [review]

File: container/snippets/quickstart.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START gke_list_cluster]
+async function main() {
+  // [START container_quickstart]
```
Comment (ace-n) [inline container/snippets/quickstart.js:19]: AFAICT, this region tag is **only** used [as a standalone sample](https://cloud.google.com/kubernetes-engine/docs/samples/container-quickstart).

I'm not sure if those are in-scope for these PRs. **If they are**, feel free to remove the _Do not merge_ label.

**Review by bourgeoisor — APPROVED** [review]
LGTM

File: container/snippets/quickstart.js (hunk 1 above)
Comment (grayside) [reply container/snippets/quickstart.js:41]: The other region tag is used, I'm trying not to modify samples unless necessary to pass tests.

**Pull request merged**